README.md

```diff
@@ -11,7 +11,7 @@ A set of tools for writing [Substrate](https://substrate.io/)-based
 [Polkadot](https://wiki.polkadot.network/en/)
 [parachains](https://wiki.polkadot.network/docs/en/learn-parachains). Refer to the included
 [overview](docs/overview.md) for architectural details, and the
-[Cumulus tutorial](https://docs.substrate.io/tutorials/v3/cumulus/start-relay) for a
+[Connect to other chains tutorials](https://docs.substrate.io/tutorials/connect-other-chains/) for a
```

Comment:
that link 404s for me?

Reply:
Just changed https://docs.substrate.io/tutorials/connect-relay-and-parachains/

Reply:
```suggestion
[Connect to relay and  parachain tutorials](https://docs.substrate.io/tutorials/connect-relay-and-parachains/) for a
```

Reply:
@lisa-parity could we put in a redirect for changes like this as I am sure the previous URL is used many places external to the docs too now.

Reply:
yeah, that makes sense.